### DISCUSSION TITLE
Wechselrichter sanft hochfahren um Kondensatoren zu schonen

### DISCUSSION THREAD

**Discussion by futzicomp** [hoylabs/OpenDTU-OnBattery#466] [Ideas]
Ich probiere es einfach mal auch hier:
Vielen Dank für das OpenDTU on Battery Projekt. Den Trockenlauf neben dem Vorhandenen OpenDTU hat mein Aufbau schon bestanden. Ich möchte ihn bald schaf stellen.
Folgende Konstellation verwende ich  momentan:
PV Power --> Victron 100/20 --> Akku --> HM-300 (Derzeit OpenDTU)  --> ShellyPM  --> Shelly3EM --> Homeassistant
Die Nulleinspeisung habe ich über NodeRed zuverlässig am laufen.
Unterhalb einer bestimmten Batteriespannung, die der Victron bereitstellt schalte ich den Wechselrichter aus und oberhalb einer bestimmten Spannung wieder ein. Mich stört dabei aber dass der Wechselrichter die ganze Nacht über 1,3W Strom verbraucht wenn ich ihn automatisiert über "turn Inverter off" abschalte. Dann kam ich auf die Idee über das BMS -ebenfalls automatisiert- den AKKU zu und abzuschalten. 
Ich glaube aber das ist keine so gute Idee?!
Vermutlich jeder hier, der mit akkus hantiert hat, hat bemerkt dass beim Anschließen des WR es kurz einen Funken gibt. - Inrush current beim schlagartigen Aufladen der WR Kondensatoren. Ich denke diesen Strompeak auf die Kondensatoren bekomme ich auch wenn ich über die Mosfets im BMS den Akkustrom einschalte????
Also muss ein "Antispark" Schalter zwischen Akku und WR?  Einen auf meine Spannungen angepassten Antispark habe ich mir gebaut : https://github.com/msglazer/Anti-Spark_Switch
Dieser Antispark fährt die Spannung innerhalb von 1,3s langsam hoch und schont die WR Kondensatoren.
Nun zur eigentlichen Frage:
Zum Schalten des Antispark benötige ich eine schaltbare Masse (Siehe Schaltplan) ABER wie realisiere ich so etwas mit dem ESP von openDTU on Battery??? Oder mach ich mir zuviele Gedanken, da mein BMS (JBD) beim Zuschalten des "Discharge Switch" evt.  bereits ab Werk die Spannung geregelt hoch fährt? Wie schafft realisiert Ihr es, dass euer Wechselrichter tatsächlich ausgeschaltet ist und keinen Strom verbraucht? Oder juckt den Wechselrichter die Stromspitze die er vom 48V Akku erhält garnicht?

Über Antworten von euch würde ich mich wahnsinnig freuen!  Ja und die Idee ist einen Masseschalter zum Schalten der Batterieschaltung zu integrieren, falls nicht schon eine andere Möglichkeit existiert ;-) 

Vielen Dank Uli 
(Nicht böse sein, wenn ich in Discord die gleiche Frage gestellt habe. Für mich alten Knochen ist das hier aber übersichtlicher, angenehmer und zielgerichteter - Sorry!)
GitHub
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/6124770/6be276bc-aba5-439e-9194-d1136f7f17a5)

**Comment by spcqike** [comment]
Du könntest auch einfach den WR auf der AC Seite schalten, oder? Zieht er dann auch noch 1.3W aus dem Akku? 

Und… 1.3W …. Das macht bei 24/7 im Jahr 3,40€. Wenn durchschnittlich 12h Nacht noch 1,70€/Jahr. 

Lohnt es sich da, was zu kaufen und zu basteln und am Ende doch zu riskieren, dass der WR beschädigt wird? 

Was du machen könntest. Ein Widerstand (10Ohm? 100Ohm?)und parallel ein bistabiles Relais bzw. ein Schrittrelais. 

Das Relais lässt du offen, sodass nach einschalten des BMS erst einmal nur über den Widerstand sanft Spannung angelegt wird. Nach ner Sekunde schaltest du das Relais und brückst damit den Widerstand. 
Bistabil damit das Relais selbst nicht dauerhaft erregt bleiben muss und selbst Strom verbraucht. 

(Alternativ ein NC Relais was zum zuschalten der Batterie vorher offen gehalten / erregt wird. Und dann nach kurzer Zeit wieder Spannungsfrei geschalten wird / abfällt.)

**Reply by futzicomp** [reply]
Meine Module leisten nur 400W und ich habe vorübergehend nur 2 ausrangierten Pedelec Akkus ( Insg ca 600Wh) installiert. Wenn ich den Akku ziemlich leersauge bewirken die 1,3W schon einen gehörigen Spannungsabfall über Nacht...
Bin erst am üben......

**Comment by swingstate** [comment]
So ein Antispark könnte ich mir auch installieren :) Wieviel Widerstand hat der denn? 

Zu Deiner Frage, mich stört das bisschen Verbrauch des WRs nicht, im Verhältnis zur Sonnenenergie die ich durch das gesamte System "ernten" kann. Da lass ich den WR lieber elektrisch verbunden, als dass ich ständig die Batterie ab und wieder zu schalte. 

Habe bisher jedoch schon öfters die Batterie mittels Trennschalter getrennt und bisher keine Probleme festgestellt. Ich schalte aber vorher immer den WR auf der AC Seite mit einem Shelly Plus aus, so dass die Kondensatoren nicht leer gesaugt werden (lt. meiner Theorie).

**Comment by Manos1966** [comment]
Hallo Leute, ich habe eine Frage:
Es gibt Batterien die einen so genannten "Soft-Start" haben (z.B. Pylontech US5000)

> Die eingebaute Soft-Start-Funktion kann den Stromschlag reduzieren,
wenn der Wechselrichter von der Batterie gestartet werden muss

Ersetzt diese Funktion die "Notwendigkeit" eines Antispark?

**Reply by kloppy1984** [reply]
der 5000er hat wohl diese Funktion nicht.
Alle Pylontech mit "C" m Ende haben den SoftStart

**Reply by Manos1966** [reply]
> Alle Pylontech mit "C" m Ende haben den SoftStart

Das gilt fuer die 2000er und 3000er Reihe.
Es gibt keine US5000C, nur US5000 und US5000-B und beide haben SoftStart (laut Handbuch).
Ersetzt also diese Funktion die "Notwendigkeit" eines Antispark?

![US5000SoftStart](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/e561d580-00ec-4c00-8ce3-0662f9427ce5)

**Comment by kloppy1984** [comment]
Ich hab diese Platine drin:
https://solarmodule-gladbeck.de/produkt/einschaltstrombegrenzung-fuer-balkonkraftwerke-mit-speicher/

Davor habe ich es per SicherungsAutomat getrennt, bisher ohne Probleme.
Viele Mppts haben doch einen Schaltbaren Ausgang?

Diese 1,3W machen den Bock nicht mehr Fett.
Einmal Fernseher oder Licht eingeschaltet, ist der Spar Effekt zunichte.

**Reply by Manos1966** [reply]
Vielen Dank fuer den Link!
ich habe zwar eine Pylontech (mit SoftStart), diese Platine gibt aber die Moeglichkeit einfache(re) Batterien einzusetzen 👍 
Vor allem jetzt, dass die OpenDTUonBattery auch mit Victron SmartShank arbeiten kann, ist es sehr interessant.

**Comment by futzicomp** [comment]
Hallo kloppy,
Danke für den Link zu deiner Platine. Werde ich mir anschauen.
Den schaltbaren Ausgang kann ich nicht verwenden, da der victron 20/100 keine (nennenswerte) Last über den Lastausgang schalten kann.
Mein Wechselrichter zeigt ein Verbrauch von 1.3 w an. Stimmt das ist natürlich wenig. Wenn die 1.3 w denn tatsächlich stimmen.??
Fakt ist, dass mein Akku nach dem Abschalten in der Spannung steigt, und dann über Nacht wieder über 2V verliert. Klar, das liegt auch  an der Cellchemie und an der geringen Kapazität (Pedelec Akku) aber so Krebse ich dauernd mit total entleerten Akku Rum, insbesondere, wenn jetzt dann bald die Leidenszeit kommt 😉

Freundliche Grüße

U. Dörr
________________________________

**Comment by Landixus** [comment]
Ich habe auch direkt dran 48V zu HM 300, der fährt mit der DTU gaaaaaanz langsam hoch.
Ich schalte AC ab 2300 Uhr dazu und ab 0700 Uhr wieder weg(Klassisch mit mechanischer Zeit Schaltuhr).
AC Lader geht an wenn mehr als 800W produziert wird(Suche noch nach Steuerung mit Tasmota, hätte das aber lieber in der DTU).
Ich habe die Batterie mit einem LS Schalter 20A Auslöse Charakteristik C abgesichert(Bedeutet bei dem geringsten Fehler schaltet der sofort weg). Man muss zwar 2x einschalten, aber dann hält er. Übrigens habe ich einen AC Lader 43.3V hochgeregelt auf 58.4V. Also falls Jemand so ein E-Bike Ladegerät hat mit Metallgehäuse, man kann die Einstellen, aber bitte machen lassen, wenn man sich nicht sicher ist wie das geht.
